### ISSUE TITLE
After ComfyUI update, comfyui-artgallery failed to load

### ISSUE THREAD

**Issue by wencxxxxxx** [ZHO-ZHO-ZHO/ComfyUI-ArtGallery#18]
After ComfyUI update, comfyui-artgallery failed to load

**Comment by nyukers** [comment]
Indeed, all style`s nodes have stopped showing selected style previews((

**Comment by iamicebomb** [comment]
check this
https://github.com/iamicebomb/ComfyUI-ArtGallery-mod-for-new-comfyui/blob/main/__init__.py

**Comment by jackyzhangjun** [comment]
仍然不行

**Comment by iamicebomb** [comment]
必须升级comfyui至当前版本，
修改一下_init.py_文件，uploadimg_js_file_path = xxx 文件路径 改成os.path.join(current_dir, '../../web/assets/index-p6KSJ2Zq.js')
wedgets_js_file_path = os.path.join(current_dir, '../../web/extensions/core/uploadImage.js'') 这句 wedgets_js_file_path = os.path.join(current_dir, '../../web/assets/index-DIU5yZe9.js')
在当前的0.30.10版本下可用,修改的文件内容是针对不同版本的comfyui的。

**Comment by iamicebomb** [comment]
其实是comfyui升级后，comfyui web的相关js文件名改了
懒了，所以没有加上搜索字符串 确定文件名的代码。
这样每次升级，再修改吧。

**Comment by HChuitailang** [comment]
> 必须升级comfyui至当前版本， 修改一下_init.py_文件，uploadimg_js_file_path = xxx文件路径改成os.path.join(current_dir, '../../web/assets/index-p6KSJ2Zq.js') wedgets_js_file_path = os.path.join(当前目录, '../../web/extensions/core/uploadImage.js'') 这句 wedgets_js_file_path = os.path.join(current_dir, '../../web/assets/index-DIU5yZe9.js') 在当前的0.30.10版本下可用，修改的文件内容是针对不同版本的comfyui的。

你说了半天完全 没看懂你在说什么，你这里引用的所有的东西在_init.py_文件里都有

**Comment by HChuitailang** [comment]
> 必须升级comfyui至当前版本， 修改一下_init.py_文件，uploadimg_js_file_path = xxx文件路径改成os.path.join(current_dir, '../../web/assets/index-p6KSJ2Zq.js') wedgets_js_file_path = os.path.join(当前目录, '../../web/extensions/core/uploadImage.js'') 这句 wedgets_js_file_path = os.path.join(current_dir, '../../web/assets/index-DIU5yZe9.js') 在当前的0.30.10版本下可用，修改的文件内容是针对不同版本的comfyui的。

你说了半天完全 没看懂你在说什么，你这里引用的所有的东西在_init.py_文件里都有

**Comment by Erquint** [comment]
@iamicebomb Sadly, none of the files you suggest pointing `__init__.py` to seem to actually exist in the installation of ComfyUI.

**Comment by mhuang001** [comment]
After I changed these lines near #55 and #265
```
..._js_file_path = os.path.join(current_dir, '../../web/assets/index-xxxx.js')
```
to
```
import pathlib
 
_ = list(pathlib.Path(current_dir).glob('../../web/assets/index-*.js'))
..._js_file_path = str(_[0])
```

I find the directory '../../web/assets/index-*.js'))' doesn't exist.

**Comment by Erquint** [comment]
Dead repo, I guess. See if anybody opens an independent fork…
